pkg/api/types.go

```diff
@@ -1873,3 +1873,11 @@ type ComponentStatusList struct {
 
 	Items []ComponentStatus `json:"items"`
 }
+
+type ServiceIPAllocation struct {
+	TypeMeta   `json:",inline"`
+	ObjectMeta `json:"metadata,omitempty"`
+
+	CIDR string `json:"cidr"`
```

Comment:
We can also in future do run length encoding on this, just haven't bothered yet.